superset/databases/commands/create.py

```diff
@@ -70,13 +71,25 @@ def run(self) -> Model:
         try:
             database = DatabaseDAO.create(self._properties, commit=False)
             database.set_sqlalchemy_uri(database.sqlalchemy_uri)
+            db.session.flush()
+
+            ssh_tunnel = None
```

Comment:
you can remove this

Reply:
we need the `ssh_tunnel` or we'll get a referenced before assigned error

Reply:
my bad, I was reading line 77 as doing the assignment, but it's a slightly diff variable name.